codespell_lib/data/dictionary.txt

```diff
@@ -12984,6 +12988,7 @@ fugured->figured
 fugures->figures
 fule->file
 fulfiled->fulfilled
+fullfil->fulfil, fulfill,
 fullfiled->fulfilled
 fullfiling->fulfilling
 fullfill->fulfill
```

Comment:
These (and below) should suggest both, in appropriate order, not -ed or -ing seemingly.

Reply:
I'm not quite sure what you mean by that remark.

Reply:
So I think you've done it, but suggest US and BE versions where they both exist and vary, with the closest typo first. I think when I looked the -ed and -ing variants don't vary by language.

Reply:
yes.